tox.ini

```diff
@@ -50,8 +52,7 @@ deps:
     sphinx
 
 [flake8]
-enable-extensions = U4
 exclude = conf.py
-hang-closing = True
 jobs = 1
 max-line-length = 79
+ignore = E123, E133, W503, W504
```

Comment:
Justification for ignoring:

* `E123` and `E133` -- these are too-strict (and mutually exclusive) interpretations of [PEP8's guideline about closing braces/brackets/parens](https://www.python.org/dev/peps/pep-0008/#indentation)

  > The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character [violates ] of the last line of list ... or it may be lined up under the first character of the line that starts the multiline construct ...

* `W503` and `W504` -- these are also too-strict (and mutually exclusive) interpretations of [PEP8's guideline about line break before/after operator](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)

  Though PEP8 _recommends_ "Knuth style" for "newer codes" (that is, following `W504`), this might be bikeshedding.